components/molecules/m-cookie-notification.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <transition name="fade" appear>
+    <div v-if="isOpen" class="m-cookie-notification">
+      <div class="cookie">
+        <div class="cookie__message">
+          {{ message }}
+          <router-link
+            :to="localizedRoute(detailsLink)"
+            :title="detailsLinkText"
+            class="cookie__message-link"
+          >
+            {{ detailsLinkText }}
+          </router-link>
+        </div>
+        <div class="cookie__icon">
+          <SfIcon
+            icon="cross"
+            size="xxs"
+            @click="accept"
+          />
+        </div>
+      </div>
+    </div>
+  </transition>
+</template>
+
+<script>
+import i18n from '@vue-storefront/i18n';
+import { SfIcon } from '@storefront-ui/vue';
+export default {
+  name: 'MCookieNotification',
+  components: { SfIcon },
+  props: {
+    detailsLinkText: {
+      type: String,
+      default: i18n.t('See details')
+    },
+    detailsLink: {
+      type: String,
+      default: '/privacy'
+    },
+    message: {
+      type: String,
+      default: i18n.t('We use cookies to give you the best shopping experience.')
+    }
+  },
+  data () {
+    return {
+      isOpen: false
+    };
+  },
+  created () {
```

Comment:
`created` is triggered in ssr and csr, but we want to do this only in csr, so maybe better hook is `beforeMount` for that action, wdyt?

Reply:
and if you will move this to beforeMount, then probably third line should be changed from 
`<div v-if="isOpen" class="m-cookie-notification">`
to 
`<div v-show="isOpen" class="m-cookie-notification">`
to avoid mismatch.

Reply:
You are right, thanks for noticing this issue.